ivy/container/general.py

```diff
@@ -3532,3 +3533,185 @@ def supports_inplace_updates(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+        
+    @staticmethod
+    def static_stable_pow(
```

Comment:
There is another function with the same name in this file, please make sure that there's only one function named `static_stable_pow`